metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -235,7 +247,7 @@ def get_datasource_urn_from_id(self, datasource_id):
         return None
 
     def construct_dashboard_from_api_data(self, dashboard_data):
-        dashboard_urn = f"urn:li:dashboard:({self.platform},{dashboard_data['id']})"
+        dashboard_urn = f"urn:li:dashboard:({self.platform},{self.platform_instance}{dashboard_data['id']})"
```

Comment:
let's use our make_chart_urn and make_dashboard_urn helpers - they also have a proper platform_instance kwargs, which should also let us avoid needing `self.platform_instance`

Reply:
Thank you for the tip :+1: 
It works fine perfectly :ok_hand: 


Example of logs:
```
 'entities': {'dashboard': ['urn:li:dashboard:(superset,prod-bedrock.508)',
                            'urn:li:dashboard:(superset,prod-bedrock.543)',
                            'urn:li:dashboard:(superset,prod-bedrock.242)',
                            'urn:li:dashboard:(superset,prod-bedrock.337)',
                            'urn:li:dashboard:(superset,prod-bedrock.509)',
                            'urn:li:dashboard:(superset,prod-bedrock.567)',
                            'urn:li:dashboard:(superset,prod-bedrock.118)',
                            'urn:li:dashboard:(superset,prod-bedrock.428)',
                            'urn:li:dashboard:(superset,prod-bedrock.340)',
                            'urn:li:dashboard:(superset,prod-bedrock.183)',
                            '... sampled of 212 total elements'],
              'chart': ['urn:li:chart:(superset,prod-bedrock.3143)',
                        'urn:li:chart:(superset,prod-bedrock.9784)',
                        'urn:li:chart:(superset,prod-bedrock.11400)',
                        'urn:li:chart:(superset,prod-bedrock.11399)',
                        'urn:li:chart:(superset,prod-bedrock.11398)',
                        'urn:li:chart:(superset,prod-bedrock.10060)',
                        'urn:li:chart:(superset,prod-bedrock.10052)',
                        'urn:li:chart:(superset,prod-bedrock.10050)',
                        'urn:li:chart:(superset,prod-bedrock.10046)',
                        'urn:li:chart:(superset,prod-bedrock.11396)',
```